public/components/datasources/components/manage/associated_objects_tab.tsx

```diff
@@ -122,40 +136,119 @@ export const AssociatedObjectsTab: React.FC<AssociatedObjectsTabProps> = ({
       field: 'createdByIntegration',
       name: 'Created by Integration',
       sortable: true,
+      render: (createdByIntegration: string, _item: AssociatedObject) =>
+        createdByIntegration ? (
+          <EuiLink onClick={() => openDetailsPage(createdByIntegration)}>
+            {createdByIntegration}
+          </EuiLink>
+        ) : (
+          '-'
+        ),
     },
     {
       field: 'accelerations',
       name: 'Accelerations',
       sortable: true,
+      render: (accelerations: string[]) => {
+        return accelerations.length > 0
+          ? accelerations.map((acceleration, index) => (
+              <React.Fragment key={index}>
+                <EuiLink onClick={() => openFlyout(acceleration)}>{acceleration}</EuiLink>
+                {index < accelerations.length - 1 ? ', ' : ''}
+              </React.Fragment>
+            ))
+          : '-';
+      },
     },
     {
       name: 'Actions',
       actions: [
         {
-          name: 'Edit',
-          description: 'Edit this object',
+          name: 'Discover',
+          description: 'Discover this object',
           type: 'icon',
           icon: 'discoverApp',
-          onClick: (item: AssociatedObject) => console.log('Edit', item),
+          onClick: (item: AssociatedObject) => console.log('Discover', item),
         },
         {
-          name: 'Delete',
-          description: 'Delete this object',
+          name: 'Accelerate',
+          description: 'Accelerate this object',
           type: 'icon',
           icon: 'bolt',
-          onClick: (item: AssociatedObject) => console.log('Delete', item),
+          available: (item: AssociatedObject) => item.type === 'Table',
+          onClick: (item: AssociatedObject) => console.log('Accelerate', item),
         },
       ],
     },
   ];
 
+  const onSearchChange = ({ query, error }) => {
+    if (error) {
+      console.log('Search error:', error);
+      return;
+    }
+
+    const matchesClauses = (obj, clauses) => {
+      if (!clauses.length) {
+        return true;
+      }
+
+      return clauses.some((clause) => {
+        if (clause.type === 'field') {
```

Comment:
We may want to consider refactor the logics here a little bit to make it clear

```
const isClauseMatched = (record: AssociatedObject, filterObj: associatedTableFilter): boolean => Object.entries(record).some(([key, value]) => key === filterObj['field'] && filterObj.value === value);

const matchesClauses = (obj, clauses) => {
  if (clauses.length === 0) return true;
  return clauses.some((clause) => {
    if (clause.type !== 'field' && clause.field !== ACCELERATION) return true;
    if (clause.field === ACCELERATION) return obj[ACCELERATION].includes(clause.value);
    return isClauseMatched(obj, clause);
  });
};
```

Reply:
fixed.